northstar/src/runtime/minijail.rs

```diff
@@ -82,19 +83,30 @@ impl Launcher for Minijail {
         })?;
         let mut lines = io::BufReader::new(async_reader).lines();
 
-        // Spawn a task that forwards logs from minijail to the rust logger.
-        let log_task = task::spawn(async move {
-            while let Ok(Some(line)) = lines.next_line().await {
-                let l = line.split_whitespace().skip(2).collect::<String>();
-                match line.chars().next() {
-                    Some('D') => debug!("{}", l),
-                    Some('I') => info!("{}", l),
-                    Some('W') => warn!("{}", l),
-                    Some('E') => error!("{}", l),
-                    _ => trace!("{}", line),
+        let stop_token = CancellationToken::new();
+        let log_task = {
+            let stop_token = stop_token.clone();
+
+            // Spawn a task that forwards logs from minijail to the rust logger.
+            task::spawn(async move {
+                loop {
+                    select! {
+                        Ok(Some(line)) = lines.next_line() => {
+                            let l = line.split_whitespace().skip(2).collect::<String>();
+                            match line.chars().next() {
+                                Some('D') => debug!("{}", l),
+                                Some('I') => info!("{}", l),
+                                Some('W') => warn!("{}", l),
+                                Some('E') => error!("{}", l),
+                                _ => trace!("{}", line),
+                            }
+                        }
+                        _ = stop_token.cancelled() => break,
+                        else => break,
+                    }
```

Comment:
Think this is unrelated to the PR.

Reply:
yes, partially, I needed this at some point to be able to stop the `tokio` runtime after the shutdown.

Reply:
makes sense